### PR TITLE
Fix #287 Servlet specific properties aren't converted

### LINKED ISSUE #287: Servlet specific properties aren't converted
##  Environment Details
* Transformer Version: 0.5.0
* JDK version: openjdk 11.0.12
* OS: Red Hat Enterprise Linux 7.4

## Problem Description
The Servlet property names listed below are not converted even if they exist in the java or class files.
* https://jakarta.ee/specifications/servlet/5.0/jakarta-servlet-spec-5.0.html#ssl-attributes
* https://jakarta.ee/specifications/servlet/5.0/jakarta-servlet-spec-5.0.html#temporary-working-directories
* https://jakarta.ee/specifications/servlet/5.0/jakarta-servlet-spec-5.0.html#included-request-parameters
* https://jakarta.ee/specifications/servlet/5.0/jakarta-servlet-spec-5.0.html#forwarded-request-parameters
* https://jakarta.ee/specifications/servlet/5.0/jakarta-servlet-spec-5.0.html#dispatched-request-parameters
* https://jakarta.ee/specifications/servlet/5.0/jakarta-servlet-spec-5.0.html#request-attributes

## Steps to reproduce

* Example: TestServlet.java
```java
package test.servlet;

import java.io.IOException;

import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

@WebServlet(urlPatterns={"/test"})
public class TestServlet extends HttpServlet {
	public void doGet(HttpServletRequest req, HttpServletResponse res) throws IOException {
		res.setContentType("text/html; charset=UTF-8");
		res.getWriter().println("<html><head><title>Test</title></head><body>");
		res.getWriter().println("Hello, World!<br/>");
		String cipherSuite = (String)req.getAttribute("javax.servlet.request.cipher_suite");
		res.getWriter().println("cipherSuite="+cipherSuite);
		res.getWriter().println("</body></html>");
	}
}
```

## Cause
This is because the property names are not written in the following file.
https://github.com/eclipse/transformer/blob/main/org.eclipse.transformer.jakarta/src/main/resources/org/eclipse/transformer/jakarta/jakarta-renames.properties

## Impact of Issue
Since the conversion of the class file is insufficient, the lack of conversion could be detected only in operation and becomes a problem.

### PR DESCRIPTION
**Pull request by 11rx4f**
* Fixes: #287

Signed-off-by: Ryosuke Okada <ryosuke.okada@fujitsu.com>

### CODE REVIEW

File: org.eclipse.transformer.jakarta/src/main/resources/org/eclipse/transformer/jakarta/jakarta-renames.properties [hunk 1]
```diff
@@ -156,3 +156,35 @@ javax.xml.ws.spi.http=jakarta.xml.ws.spi.http
 javax.xml.ws.spi=jakarta.xml.ws.spi
 javax.xml.ws.wsaddressing=jakarta.xml.ws.wsaddressing
 javax.xml.ws=jakarta.xml.ws
+
+# Servlet Properties
```
Comment (bjhargrave) [inline org.eclipse.transformer.jakarta/src/main/resources/org/eclipse/transformer/jakarta/jakarta-renames.properties:160]: I don't think property transform rules should be in jakarta-renames which is for package transforms. I think these should instead be in jakarta-direct which is about string constant transforms. Can you please move them to jakarta-direct?

**Review by bjhargrave — CHANGES_REQUESTED** [review]

File: org.eclipse.transformer.jakarta/src/main/resources/org/eclipse/transformer/jakarta/jakarta-renames.properties (hunk 1 above)
Comment (11rx4f) [reply org.eclipse.transformer.jakarta/src/main/resources/org/eclipse/transformer/jakarta/jakarta-renames.properties:160]: They are already exists in jakarta-direct so class file is transformed properly.
but, java file is not converted.

Comment (bjhargrave) [reply org.eclipse.transformer.jakarta/src/main/resources/org/eclipse/transformer/jakarta/jakarta-renames.properties:160]: OK, when #186 is fixed by #283, then direct rules will be applied to Java files. So I don't want to duplicate the direct rules in the rename rules.

**Comment by bjhargrave** [comment]
I picked up all the changes you proposed here in #309 which covered more then servlet. #309 updated the direct rules which will be used to transform Java files once #283 is merged.

**Pull request closed without merging**